test/functorch/common_utils.py

```diff
@@ -268,18 +270,46 @@ def f(dummy, *args, **kwargs):
     outer_in_dims = (0,) + in_dims
     batched_args, kwarg_values = maybe_clone_inputs()
     output = vmap(vmap(f, inner_in_dims), outer_in_dims)(dummy, *batched_args, **kwarg_values)
-    yield (expected, output)
+
+    yield (loop_out, batched_out, expected, output)
```

Comment:
Nit: I'd name `expected` and `output` something more descriptive now (vmapvmap_expected, vmapvmap_output) or something.

Also, the order appears to be reversed, we should be returning (output, expected) (the convention for checking assertions is assertEquals(output, expected). Seems like the original code was wrong.